### ISSUE TITLE
[Suggestion] Modify Vacuum Tube Assembler recipes to allow some use before EBF.

### ISSUE THREAD

**Issue by DukeGrimm** [GTNewHorizons/NewHorizons#2188]
#### Which modpack version are you using?
2.0.1.3
#
#### If in multiplayer; On which server does this happen? 
SP
#
#### What did you try to do, and what did you expect to happen?
Making Vacuum tubes in assembler before EBF, I expected to be able to craft the basic 1 output recipe for vacuum tubes. I had thought I would be able to fluid extract Red Alloy for the recipe with my LV Fluid Extractor.
#
#### What happend instead? (Attach screenshots if needed)
I was unable to use the Assembler recipe because the required Molten Red Alloy requires an MV fluid extractor, and the Molten Red stone Alloy can be liquefied in LV extractor but requires EBF. Basically you are stuck with hand crafting vacuum tubes until EBF to make Redstone Alloy. This also makes the recipies using Red Alloy obsolete, since by the time you can liquefy red alloy, you can make Redstone Alloy and Annealed Copper. 
#
#### What do you suggest instead/what changes do you propose?
Options:
1.) Make it possible to melt "Red Alloy" in LV Fluid Extractor (I.e 32v or less.). Make "Redstone Alloy" Require MV Fluid Extractor to melt. This is will encourage people to get MV fluid Extractor for making cheap vacuum tubes, and all around make the current assembly recipes for Vacuum tubes make more sense in regards to tiers.
2.) Modify Red Alloy Vacuum tube assembler recipes, Have them require "Molten Redstone" instead. It could require a full dust per recipe (144mb)

**Comment by DukeGrimm** [comment]
I did some testing locally. I was able to change the melting points of the materials in the Material Properties.cfg in the gregtech config folder. I just swapped Redstone Alloy's melting point and Red Alloy's melting point to get "option 1" working.

I think the last big question for "option 1", would be if there is a particular balance reason why Red Alloy wasn't melt-able at LV level.

Old:
```
redalloy{
            I:MeltingPoint_671=671
}
redstonealloy{
            I:MeltingPoint_500=500
}
```
New:
```
redalloy{
            I:MeltingPoint_671=500
}
redstonealloy{
            I:MeltingPoint_500=671
}
```
